### ISSUE TITLE
Bugzilla description partially discarded

### ISSUE THREAD

**Issue by MoonlightSentinel** [dlang/dlang-bot#236]
Dlang-Bot listed an incomplete description in the Bugzilla section of dlang/phobos#7376:

![IncompleteDescription](https://user-images.githubusercontent.com/26939771/73305334-2bd23200-421a-11ea-8d6c-c60e1b161bff.jpg)

The real description was:

```
Issue 20540 - (White|Black)Hole does not work with return|scope functions
```

HTML:

```
<td>(White</td>
```

**Comment by wilzbach** [comment]
I couldn't replicate this locally, so I added a test for this to the testsuite: https://github.com/dlang/dlang-bot/pull/257

**Comment by wilzbach** [comment]
Fixed by #257
